build-push-docker-image.sh

```diff
@@ -6,5 +6,10 @@ MODULE_DIR=${3}
 
 echo "building version $MODULE_VERSION of module $MODULE_NAME in $MODULE_DIR"
 
-docker build -t coatrack/$MODULE_NAME:$MODULE_VERSION --build-arg MODULE_VERSION=$MODULE_VERSION --build-arg MODULE_NAME=$MODULE_NAME --build-arg MODULE_DIR=$MODULE_DIR .
+if [ "$MODULE_NAME" == "coatrack-admin" ]; then
+  docker build -t coatrack/$MODULE_NAME:$MODULE_VERSION --build-arg MODULE_VERSION=$MODULE_VERSION --build-arg MODULE_NAME=$MODULE_NAME --build-arg MODULE_DIR=$MODULE_DIR -f ./Dockerfile-Admin .
```

Comment:
It is hard to spot the difference between if/else. The only difference seems to be the Dockerfile(?!), so it would be easier to read if the if/else would just set a variable with the dockerfile name, and then the build command is only written once.

Anyway, maybe this is not needed at all?! See comment related to the Dockerfile.

Reply:
Done. See comment above.